keps/sig-auth/0014-dynamic-audit-configuration.md

```diff
@@ -182,6 +207,10 @@ It may also be reasonable to provide a dynamic auth configuration from secrets,
 
 This needs further discussion.
 
+#### Performance
+
+These changes will likely have an O(n) performance impact on the api server per policy.  A `DeepCopy` of the event will be required for each backend. Also, the request/response object would now be serialized on every [request](https://github.com/kubernetes/kubernetes/blob/cef2d325ee1be894e883d63013f75cfac5cb1246/staging/src/k8s.io/apiserver/pkg/audit/request.go#L150-L152). Benchmark testing will be required to understand the scope of the impact and what optimizations may be required. This impact is gated by opt-in feature flags, which largely mitigates the concern.
```

Comment:
> This impact is gated by opt-in feature flags, which largely mitigates the concern.

I disagree with this statement. I think it unblocks the alpha implementation & lets us start benchmarking, but it is still a concern that needs to be addressed (or dismissed with data) before going to beta.